backend_new/src/services/translation.service.ts

```diff
@@ -0,0 +1,167 @@
+import { Injectable } from '@nestjs/common';
+import { PrismaService } from './prisma.service';
+import { LanguagesEnum, Prisma } from '@prisma/client';
+import { ListingGet } from '../dtos/listings/listing-get.dto';
+import { GoogleTranslateService } from './google-translate.service';
+import * as lodash from 'lodash';
+
+@Injectable()
+export class TranslationService {
+  constructor(
+    private prisma: PrismaService,
+    private readonly googleTranslateService: GoogleTranslateService,
+  ) {}
+
+  public async getTranslationByLanguageAndJurisdictionOrDefaultEn(
+    language: LanguagesEnum,
+    jurisdictionId: string | null,
+  ) {
+    try {
+      return await this.prisma.translations.findUniqueOrThrow({
```

Comment:
how do you feel about switching this to `findUnique` and then we handle the error ourselves instead of wrapping a try catch?

Reply:
`findUnique` looks a lot cleaner!